tests/Form/InputPasswordTest.php

```
@@ -0,0 +1,622 @@
+        self::assertSame(
+            <<<HTML
+            <input id="inputpassword-" type="password" form="form-id">
+            HTML,
+            InputPassword::tag()->form('form-id')->id('inputpassword-')->render(),
+            "Failed asserting that element renders correctly with 'form' attribute.",
+        );
+    }
+
+    public function testRenderWithGlobalDefaultsAreApplied(): void
+    {
+        SimpleFactory::setDefaults(InputPassword::class, ['class' => 'default-class']);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\UIAwesome\Html\Core\Factory\SimpleFactory' in method 'testRenderWithGlobalDefaultsAreApplied'. (undefined)

(StaticAccess)